content/en/docs/reference/command-line-tools-reference/feature-gates/watch-list-client.md

```diff
@@ -0,0 +1,16 @@
+---
+title: WatchListClient
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
+    defaultValue: false
+    fromVersion: "1.30"
+---
+Enable the API server to serve collections (responses to the **list** verb)
+from its watch cache, rather than always from the back store etcd.
+When this feature gate is disabled, the API server falls back to its legacy behavior
+where data are fetched from etcd instead of served from the watch cache.
```

Comment:
OK, this is wrong; see https://github.com/kubernetes/website/pull/47261/files#r1692893760